ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -322,6 +317,13 @@ function run {
     else
         echo "${failures} failures in openshift-extended-test" | tee -a "${SHARED_DIR}/openshift-e2e-test-qe-report-openshift-extended-test-failures"
     fi
+    # it ensure the the step after this step in test will be executed per https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
+    # please refer to the junit result for case result, not depends on step result.
+    if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
+        echo "do not force success exit"
+        exit $ret_value
+    fi
+    return 0
```

Comment:
@aleskandro it works, but if we align code with other step, could remove this line.
it is up to you to keep it or not.

Reply:
yeah, let me do it in a while... There should be even a few others actually: I'd prefer keeping `return` statements vs `exit` in functions... But it could require some time to verify compliance with the current logic.

Reply:
ok to keep it. thanks for the information.

Reply:
I meant that I prefer having the return statements vs exit within functions when possible... But to do that change consistently, we need a few more time to avoid unexpected behavior in the test configs. 

I removed the `return 0` for now, as you recommended.